src/components/CreateCategoryForm.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import Button from "./Button";
+import Radio from "./Radio";
+import Input from "./Input";
+
+const Form = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 1fr 0.3fr;
+
+  img {
+    border-radius: 10px;
+    margin: 3%;
+    width: 400px;
+  }
+
+  .name {
+    grid-area: 2 / 1 / 3 / 3;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    label {
+      color: ${({ theme }) => theme.text.sub};
+      font-size: ${({ theme }) => theme.fontSizes.small};
+      padding-top: 10px;
+    }
+
+    input {
+      width: 5rem;
+      text-align: center;
+    }
+  }
+`;
+
+function CreateCategoryForm({ onSubmit }) {
+  const [selectedType, setSelectedType] = useState("grid");
+  const [category, setCategory] = useState("");
+
+  const handleSubmit = function () {
+    onSubmit({ categoryType: selectedType, category });
+  };
+
+  const handleSelectType = function (type) {
+    setSelectedType(type);
+  };
+
+  const handleNameChange = function ({ target }) {
+    setCategory(target.value);
+  };
+
+  return (
+    <Form>
+      <div>
+        <Radio
+          id={"customType1"}
```

Comment:
```suggestion
          id="customType1"
```
string 프로퍼티라면 {} 없이 바로 넣어주어도 동일하게 동작할 것 같은데요, 혹시 추가된 prop을 기존 존재하는 속성과 구분해주기 위해서라면, 현재 코드스타일이 통일되어있지 않은 부분이 있어서 논의 후 한쪽으로 통일할 필요가 있을 것 같습니다.

Reply:
{} 없는 쪽으로 수정하도록 하겠습니다!